lua/terrortown/menus/gamemode/equipment/base_equipment.lua

```diff
@@ -99,4 +99,14 @@ function CLGAMEMODESUBMENU:Populate(parent)
 
 		masterRefs[key] = option
 	end
+
+	-- now add custom equipment settings
+	if not isfunction(equipment.AddToSettingsMenu) then
+		print(equipment.PrintName)
+
+		PrintTable(equipment)
```

Comment:
Maybe some info about these prints? 

Eg. print something like "Ohh not adding equip to settings cause: ...." 
Maybe even omit it?